classes/install.yaml

```diff
@@ -116,22 +110,19 @@ packageScript: |
     # Only copy shared libraries.
     installPackageLib()
     {
-        if [[ ${INSTALL_SHARED:-} ]] ; then
-            installCopy "$@" /usr/ /usr/lib/ \
-                "/usr/lib/*.so.*" \
-                "/usr/lib/lib*-*.so" \
-                "!*"
-            installStripAll .
-        fi
+        installCopy "$@" /usr/ \
+            /usr/lib/ "/usr/lib/*.so*" \
+            /usr/bin/ "/usr/bin/*.dll" "/usr/bin/*.pdb" \
```

Comment:
`installPackageLib` should only install shared libraries. By removing `*.so.*` versioned libraries are broken.

Reply:
no idea, why i changed this lines handling so linux library files, doesn't depends to the windows cmake stuff. xD. I will revert this.